config/locales/errors/en.yml

```diff
@@ -9,12 +9,13 @@ en:
     messages:
       already_confirmed: was already confirmed, please try signing in
       blank: Please fill in this field.
+      confirmation_code_incorrect: Incorrect code. Did you type it in correctly?
       confirmation_invalid_token: >
-        Invalid confirmation link. Either the link expired or you 
-        already confirmed your account. 
+        Invalid confirmation link. Either the link expired or you
+        already confirmed your account.
       confirmation_period_expired: >
         Expired confirmation link.
-        You can click "Resend confirmation instructions" to get another one. 
+        You can click "Resend confirmation instructions" to get another one.
```

Comment:
I recently learned about a neat trick where you can reference an existing translation inside another one. Since "Resend confirmation instructions" is defined in `idv.messages.usps.send_again`, you can refer to it by add a colon before it, like this:
```yaml
You can click :idv.messages.usps.send_again to get another one.
```
Magic!